server/jetstream_api.go

```diff
@@ -3094,6 +3101,13 @@ func (s *Server) jsMsgGetRequest(sub *subscription, c *client, _ *Account, subje
 		return
 	}
 
+	// This version does not support batch.
```

Comment:
Is there a reason or just a choice?

Reply:
That is current behavior if you sent in a wildcard, but I would look to see. However this is also tied to the multi-get conversation in other issue. I feel that would be applicable and that here we should only do one.

For instance even with one wildcard key, this really is a multi-get with up to being last sequence of the stream when I receive the request.